Fire Stick crash while playback

### Describe the bug

Simply play any H.264(X264) 30 FPS AAC (avcodec), Stero AC3 Passthru with Chapter markers and there is a crash at the chapter marker. Log included. This happens on both ExoPlayer and LibVLC

same file plays on Windows Jellyfin client, ipad client and VLC media player

### Logs

```shell
---
client: Jellyfin for Android TV
client_version: 0.16.0
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 

java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3468)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3626)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2081)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:223)
	at android.app.ActivityThread.main(ActivityThread.java:7719)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
	at org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>(Unknown Source:12)
	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.initActions(CustomPlaybackTransportControlGlue.java:197)
	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.<init>(CustomPlaybackTransportControlGlue.java:106)
	at org.jellyfin.androidtv.ui.playback.overlay.LeanbackOverlayFragment.onCreate(LeanbackOverlayFragment.java:32)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1989)
	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1967)
	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.onCreate(CustomPlaybackOverlayFragment.java:174)
	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:252)
	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:219)
	at org.jellyfin.androidtv.ui.browsing.MainActivity.onCreate(MainActivity.kt:55)
	at android.app.Activity.performCreate(Activity.java:8019)
	at android.app.Activity.performCreate(Activity.java:8002)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3441)
	... 11 more

  
***Logcat***: 
```log
12-15 22:49:27.219 W/lyfin.androidt(28272): Accessing hidden method Ljava/util/Collections$SynchronizedCollection;-><init>(Ljava/util/Collection;Ljava/lang/Object;)V (greylist-max-o, reflection, denied)
12-15 22:49:27.221 I/TetheringManager(28272): registerTetheringEventCallback:org.jellyfin.androidtv
12-15 22:49:27.244 I/LogInitializer(28272): Debug tree planted
12-15 22:49:27.292 I/MigrationContext(28272): Requested migration from 2 to 2. Found 1 migrations in total.
12-15 22:49:27.311 W/lyfin.androidt(28272): Accessing hidden method Landroid/graphics/ImageDecoder;->createSource([B)Landroid/graphics/ImageDecoder$Source; (greylist-max-o, linking, denied)
12-15 22:49:27.324 I/SessionRepositoryImpl(28272): Restoring session
12-15 22:49:27.340 I/org.jellyfin.sdk.api.client.KtorClient(28272): GET http://10.10.0.5:8096/Branding/Configuration
12-15 22:49:27.408 I/MigrationContext(28272): Requested migration from 7 to 7. Found 1 migrations in total.
12-15 22:49:27.412 I/ActivityThemeExtensionsKt(28272): Applying theme DARK
--------- beginning of crash
12-15 22:49:27.502 E/AndroidRuntime(28272): FATAL EXCEPTION: main
12-15 22:49:27.502 E/AndroidRuntime(28272): Process: org.jellyfin.androidtv, PID: 28272
12-15 22:49:27.502 E/AndroidRuntime(28272): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3468)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3626)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2081)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.os.Looper.loop(Looper.java:223)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at java.lang.reflect.Method.invoke(Native Method)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
12-15 22:49:27.502 E/AndroidRuntime(28272): Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>(Unknown Source:12)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.initActions(CustomPlaybackTransportControlGlue.java:197)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.<init>(CustomPlaybackTransportControlGlue.java:106)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.playback.overlay.LeanbackOverlayFragment.onCreate(LeanbackOverlayFragment.java:32)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1989)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1967)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.onCreate(CustomPlaybackOverlayFragment.java:174)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:252)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:219)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.onCreate(MainActivity.kt:55)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.Activity.performCreate(Activity.java:8019)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.Activity.performCreate(Activity.java:8002)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3441)
12-15 22:49:27.502 E/AndroidRuntime(28272): 	... 11 more
12-15 22:49:27.503 E/ACRA    (28272): ACRA caught a RuntimeException for org.jellyfin.androidtv
12-15 22:49:27.503 E/ACRA    (28272): java.lang.RuntimeException: Unable to start activity ComponentInfo{org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3468)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3626)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:85)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2081)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.os.Handler.dispatchMessage(Handler.java:106)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.os.Looper.loop(Looper.java:223)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.ActivityThread.main(ActivityThread.java:7719)
12-15 22:49:27.503 E/ACRA    (28272): 	at java.lang.reflect.Method.invoke(Native Method)
12-15 22:49:27.503 E/ACRA    (28272): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:592)
12-15 22:49:27.503 E/ACRA    (28272): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:947)
12-15 22:49:27.503 E/ACRA    (28272): Caused by: java.lang.NullPointerException: Parameter specified as non-null is null: method org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>, parameter playbackController
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.playback.overlay.action.PlaybackSpeedAction.<init>(Unknown Source:12)
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.initActions(CustomPlaybackTransportControlGlue.java:197)
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.playback.overlay.CustomPlaybackTransportControlGlue.<init>(CustomPlaybackTransportControlGlue.java:106)
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.playback.overlay.LeanbackOverlayFragment.onCreate(LeanbackOverlayFragment.java:32)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.Fragment.restoreChildFragmentState(Fragment.java:1989)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.Fragment.onCreate(Fragment.java:1967)
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.playback.CustomPlaybackOverlayFragment.onCreate(CustomPlaybackOverlayFragment.java:174)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.Fragment.performCreate(Fragment.java:3094)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStateManager.create(FragmentStateManager.java:504)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:268)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:114)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1455)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:3034)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentManager.dispatchCreate(FragmentManager.java:2941)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentController.dispatchCreate(FragmentController.java:252)
12-15 22:49:27.503 E/ACRA    (28272): 	at androidx.fragment.app.FragmentActivity.onCreate(FragmentActivity.java:219)
12-15 22:49:27.503 E/ACRA    (28272): 	at org.jellyfin.androidtv.ui.browsing.MainActivity.onCreate(MainActivity.kt:55)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.Activity.performCreate(Activity.java:8019)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.Activity.performCreate(Activity.java:8002)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1309)
12-15 22:49:27.503 E/ACRA    (28272): 	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3441)
12-15 22:49:27.503 E/ACRA    (28272): 	... 11 more

```  

### App information

***App version***: `0.16.0` (`160099`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"karat","BOOTLOADER":"unknown","BRAND":"Amazon","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"karat","DISPLAY":"RS8102.1967N","FINGERPRINT":"Amazon\/karat\/karat:11\/RS8102.1967N\/0021978656644:user\/amz-p,release-keys","HARDWARE":"mt8696","HOST":"c5-ri-1804-use1a-b-18-04-fos8-24","ID":"RS8102.1967N","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"Amazon","MODEL":"AFTKM","PERMISSIONS_REVIEW_REQUIRED":true,"PRODUCT":"karat","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"amz-p,release-keys","TIME":1701543841000,"TYPE":"user","UNKNOWN":"unknown","USER":"build","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"RP1A.200720.009","CODENAME":"REL","INCREMENTAL":"0021978656644","PREVIEW_SDK_FINGERPRINT":"REL","PREVIEW_SDK_INT":0,"RELEASE":"11","RELEASE_OR_CODENAME":"11","SDK":"30","SDK_INT":30,"SECURITY_PATCH":"2020-09-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":160099,"VERSION_NAME":"0.16.0"}
```  

### Device information

***Android version***: `11`  
***Device brand***: `Amazon`  
***Device product***: `karat`  
***Device model***: `AFTKM`  

### Crash information

***Start time***: `2023-12-15T22:49:27.169-08:00`  
***Crash time***: `2023-12-15T22:49:27.554-08:00`
```


### Application version

0.16.0

### Where did you install the app from?

Amazon Appstore

### Device information

Amazon karat AFTKM

### Android version

Android 11

### Jellyfin server version

10.8.13

### Which video player implementations does this bug apply to?

- [ ] Web player (default)
- [X] Integrated player (ExoPlayer)
- [ ] External player (VLC, mpv, MX Player)

--- a/app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/playback/CustomPlaybackOverlayFragment.java
@@ -127,7 +127,6 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     private List<ChannelInfoDto> mAllChannels;
     private String mFirstFocusChannelId;
 
-    private PlaybackController mPlaybackController;
     private List<org.jellyfin.sdk.model.api.BaseItemDto> mItemsToPlay;
 
     private Animation fadeOut;
@@ -195,7 +194,6 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         int mediaPosition = videoQueueManager.getValue().getCurrentMediaPosition();
 
         playbackControllerContainer.getValue().setPlaybackController(new PlaybackController(mItemsToPlay, this, mediaPosition));
-        mPlaybackController = playbackControllerContainer.getValue().getPlaybackController();
 
         // setup fade task
         mHideTask = () -> {
@@ -350,7 +348,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         int startPos = getArguments().getInt("Position", 0);
 
         // start playing
-        mPlaybackController.play(startPos);
+        playbackControllerContainer.getValue().getPlaybackController().play(startPos);
         leanbackOverlayFragment.updatePlayState();
 
     }
@@ -358,7 +356,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     private void prepareOverlayFragment() {
         leanbackOverlayFragment = (LeanbackOverlayFragment) getChildFragmentManager().findFragmentById(R.id.leanback_fragment);
         if (leanbackOverlayFragment != null) {
-            leanbackOverlayFragment.initFromView(mPlaybackController, this);
+            leanbackOverlayFragment.initFromView(this);
             leanbackOverlayFragment.mediaInfoChanged();
             leanbackOverlayFragment.setOnKeyInterceptListener(keyListener);
         }
@@ -403,7 +401,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         public void onAudioFocusChange(int focusChange) {
             switch (focusChange) {
                 case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT:
-                    mPlaybackController.pause();
+                    playbackControllerContainer.getValue().getPlaybackController().pause();
                     break;
                 case AudioManager.AUDIOFOCUS_LOSS:
                     // We don't do anything here on purpose
@@ -424,7 +422,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                 switch (rowItem.getBaseRowType()) {
                     case Chapter:
                         Long start = rowItem.getChapterInfo().getStartPositionTicks() / 10000;
-                        mPlaybackController.seek(start);
+                        playbackControllerContainer.getValue().getPlaybackController().seek(start);
                         hidePopupPanel();
                         break;
                 }
@@ -445,7 +443,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                 leanbackOverlayFragment.hideOverlay();
 
                 // also close this if live tv
-                if (mPlaybackController.isLiveTv()) hide();
+                if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) hide();
             } else if (mGuideVisible) {
                 hideGuide();
             } else {
@@ -490,19 +488,19 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
             }
 
             if (keyCode == KeyEvent.KEYCODE_MEDIA_PLAY) {
-                mPlaybackController.play(0);
+                playbackControllerContainer.getValue().getPlaybackController().play(0);
                 return true;
             } else if (keyCode == KeyEvent.KEYCODE_MEDIA_PAUSE) {
-                mPlaybackController.pause();
+                playbackControllerContainer.getValue().getPlaybackController().pause();
                 return true;
             } else if (keyCode == KeyEvent.KEYCODE_MEDIA_PLAY_PAUSE) {
-                mPlaybackController.playPause();
+                playbackControllerContainer.getValue().getPlaybackController().playPause();
                 return true;
             } else if (keyCode == KeyEvent.KEYCODE_MEDIA_FAST_FORWARD || keyCode == KeyEvent.KEYCODE_BUTTON_R1 || keyCode == KeyEvent.KEYCODE_BUTTON_R2) {
-                mPlaybackController.fastForward();
+                playbackControllerContainer.getValue().getPlaybackController().fastForward();
                 return true;
             } else if (keyCode == KeyEvent.KEYCODE_MEDIA_REWIND || keyCode == KeyEvent.KEYCODE_BUTTON_L1 || keyCode == KeyEvent.KEYCODE_BUTTON_L2) {
-                mPlaybackController.rewind();
+                playbackControllerContainer.getValue().getPlaybackController().rewind();
                 return true;
             }
         }
@@ -559,7 +557,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                         leanbackOverlayFragment.hideOverlay();
 
                         // also close this if live tv
-                        if (mPlaybackController.isLiveTv()) hide();
+                        if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) hide();
                         return true;
                     } else if (mGuideVisible) {
                         hideGuide();
@@ -567,7 +565,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                     }
                 }
 
-                if (mPlaybackController.isLiveTv() && !mPopupPanelVisible && !mGuideVisible && keyCode == KeyEvent.KEYCODE_DPAD_DOWN) {
+                if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv() && !mPopupPanelVisible && !mGuideVisible && keyCode == KeyEvent.KEYCODE_DPAD_DOWN) {
                     if (!leanbackOverlayFragment.isControlsOverlayVisible()) {
                         leanbackOverlayFragment.setShouldShowOverlay(false);
                         leanbackOverlayFragment.hideOverlay();
@@ -596,7 +594,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                     }
                 }
 
-                if (mPlaybackController.isLiveTv() && keyCode == KeyEvent.KEYCODE_MENU || keyCode == KeyEvent.KEYCODE_BUTTON_Y) {
+                if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv() && keyCode == KeyEvent.KEYCODE_MENU || keyCode == KeyEvent.KEYCODE_BUTTON_Y) {
                     showGuide();
                     return true;
                 }
@@ -612,7 +610,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                         // up or down should close panel
                         if (keyCode == KeyEvent.KEYCODE_DPAD_DOWN || keyCode == KeyEvent.KEYCODE_DPAD_UP) {
                             hidePopupPanel();
-                            if (mPlaybackController.isLiveTv()) hide(); //also close this if live tv
+                            if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) hide(); //also close this if live tv
                             return true;
                         } else {
                             return false;
@@ -620,22 +618,22 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                     }
 
                     // Control fast forward and rewind if overlay hidden and not showing live TV
-                    if (!mPlaybackController.isLiveTv()) {
+                    if (!playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) {
                         if (keyCode == KeyEvent.KEYCODE_MEDIA_FAST_FORWARD || keyCode == KeyEvent.KEYCODE_BUTTON_R1 || keyCode == KeyEvent.KEYCODE_BUTTON_R2) {
-                            mPlaybackController.fastForward();
+                            playbackControllerContainer.getValue().getPlaybackController().fastForward();
                             setFadingEnabled(true);
                             return true;
                         }
 
                         if (keyCode == KeyEvent.KEYCODE_MEDIA_REWIND || keyCode == KeyEvent.KEYCODE_BUTTON_L1 || keyCode == KeyEvent.KEYCODE_BUTTON_L2) {
-                            mPlaybackController.rewind();
+                            playbackControllerContainer.getValue().getPlaybackController().rewind();
                             setFadingEnabled(true);
                             return true;
                         }
                     }
 
                     if (!mIsVisible) {
-                        if (!mPlaybackController.isLiveTv()) {
+                        if (!playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) {
                             if (keyCode == KeyEvent.KEYCODE_DPAD_RIGHT) {
                                 setFadingEnabled(true);
                                 return true;
@@ -648,10 +646,10 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                         }
 
                         if ((keyCode == KeyEvent.KEYCODE_DPAD_CENTER || keyCode == KeyEvent.KEYCODE_ENTER)
-                                && mPlaybackController.canSeek()) {
+                                && playbackControllerContainer.getValue().getPlaybackController().canSeek()) {
                             // if the player is playing and the overlay is hidden, this will pause
                             // if the player is paused and then 'back' is pressed to hide the overlay, this will play
-                            mPlaybackController.playPause();
+                            playbackControllerContainer.getValue().getPlaybackController().playPause();
                             return true;
                         }
 
@@ -688,12 +686,12 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
 
     public void switchChannel(String id, boolean hideGuide) {
         if (Utils.isEmpty(id)) return;
-        if (mPlaybackController.getCurrentlyPlayingItem().getId().equals(id)) {
+        if (playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem().getId().equals(id)) {
             // same channel, just dismiss overlay
             if (hideGuide)
                 hideGuide();
         } else {
-            mPlaybackController.stop();
+            playbackControllerContainer.getValue().getPlaybackController().stop();
             if (hideGuide)
                 hideGuide();
             apiClient.getValue().GetItemAsync(id, KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue().getId().toString(), new LifecycleAwareResponse<BaseItemDto>(getLifecycle()) {
@@ -703,8 +701,8 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
 
                     List<org.jellyfin.sdk.model.api.BaseItemDto> items = new ArrayList<org.jellyfin.sdk.model.api.BaseItemDto>();
                     items.add(ModelCompat.asSdk(response));
-                    mPlaybackController.setItems(items);
-                    mPlaybackController.play(0);
+                    playbackControllerContainer.getValue().getPlaybackController().setItems(items);
+                    playbackControllerContainer.getValue().getPlaybackController().play(0);
                 }
 
                 @Override
@@ -729,7 +727,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         super.onResume();
 
         // Close player when resuming without a valid playback contoller
-        if (!mPlaybackController.hasFragment()) {
+        if (!playbackControllerContainer.getValue().getPlaybackController().hasFragment()) {
             if (navigationRepository.getValue().getCanGoBack()) {
                 navigationRepository.getValue().goBack();
             } else {
@@ -774,9 +772,9 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
 
         // end playback from here if this fragment belongs to the current session.
         // if it doesn't, playback has already been stopped elsewhere, and the references to this have been replaced
-        if (mPlaybackController != null && mPlaybackController.getFragment() == this) {
+        if (playbackControllerContainer.getValue().getPlaybackController() != null && playbackControllerContainer.getValue().getPlaybackController().getFragment() == this) {
             Timber.d("this fragment belongs to the current session, ending it");
-            mPlaybackController.endPlayback();
+            playbackControllerContainer.getValue().getPlaybackController().endPlayback();
         }
 
         // Reset display mode back to "no preference"
@@ -812,7 +810,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         hide();
         leanbackOverlayFragment.setShouldShowOverlay(false);
         leanbackOverlayFragment.hideOverlay();
-        mPlaybackController.mVideoManager.contractVideo(Utils.convertDpToPixel(requireContext(), 300));
+        playbackControllerContainer.getValue().getPlaybackController().mVideoManager.contractVideo(Utils.convertDpToPixel(requireContext(), 300));
         tvGuideBinding.getRoot().setVisibility(View.VISIBLE);
         mGuideVisible = true;
         Calendar now = Calendar.getInstance();
@@ -831,7 +829,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
 
     private void hideGuide() {
         tvGuideBinding.getRoot().setVisibility(View.GONE);
-        mPlaybackController.mVideoManager.setVideoFullSize(true);
+        playbackControllerContainer.getValue().getPlaybackController().mVideoManager.setVideoFullSize(true);
         mGuideVisible = false;
     }
 
@@ -904,7 +902,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
             Timber.d("*** Display programs pre-execute");
             tvGuideBinding.channels.removeAllViews();
             tvGuideBinding.programRows.removeAllViews();
-            mFirstFocusChannelId = mPlaybackController.getCurrentlyPlayingItem().getId().toString();
+            mFirstFocusChannelId = playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem().getId().toString();
 
             if (mCurrentDisplayChannelStartNdx > 0) {
                 // Show a paging row for channels above
@@ -1238,7 +1236,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
         mHandler.postDelayed(() -> {
             if (!getLifecycle().getCurrentState().isAtLeast(Lifecycle.State.STARTED)) return;
 
-            int ndx = getCurrentChapterIndex(mPlaybackController.getCurrentlyPlayingItem(), mPlaybackController.getCurrentPosition() * 10000);
+            int ndx = getCurrentChapterIndex(playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem(), playbackControllerContainer.getValue().getPlaybackController().getCurrentPosition() * 10000);
             if (ndx > 0) {
                 mPopupRowPresenter.setPosition(ndx);
             }
@@ -1303,7 +1301,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                         if (!getActive()) return;
 
                         Utils.showToast(requireContext(), R.string.msg_recording_cancelled);
-                        mPlaybackController.updateTvProgramInfo();
+                        playbackControllerContainer.getValue().getPlaybackController().updateTvProgramInfo();
                         TvManager.forceReload();
                     }
 
@@ -1321,7 +1319,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                         if (!getActive()) return;
 
                         Utils.showToast(requireContext(), R.string.msg_recording_cancelled);
-                        mPlaybackController.updateTvProgramInfo();
+                        playbackControllerContainer.getValue().getPlaybackController().updateTvProgramInfo();
                         TvManager.forceReload();
                     }
 
@@ -1351,7 +1349,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                                 if (!getActive()) return;
 
                                 Utils.showToast(requireContext(), R.string.msg_set_to_record);
-                                mPlaybackController.updateTvProgramInfo();
+                                playbackControllerContainer.getValue().getPlaybackController().updateTvProgramInfo();
                                 TvManager.forceReload();
                             }
 
@@ -1369,7 +1367,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                                 if (!getActive()) return;
 
                                 Utils.showToast(requireContext(), R.string.msg_set_to_record);
-                                mPlaybackController.updateTvProgramInfo();
+                                playbackControllerContainer.getValue().getPlaybackController().updateTvProgramInfo();
                                 TvManager.forceReload();
                             }
 
@@ -1409,7 +1407,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     }
 
     public void updateDisplay() {
-        org.jellyfin.sdk.model.api.BaseItemDto current = mPlaybackController.getCurrentlyPlayingItem();
+        org.jellyfin.sdk.model.api.BaseItemDto current = playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem();
         if (current != null && getContext() != null) {
             leanbackOverlayFragment.mediaInfoChanged();
             leanbackOverlayFragment.onFullyInitialized();
@@ -1437,7 +1435,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
                 binding.itemTitle.setVisibility(View.VISIBLE);
             }
 
-            if (mPlaybackController.isLiveTv()) {
+            if (playbackControllerContainer.getValue().getPlaybackController().isLiveTv()) {
                 prepareChannelAdapter();
             } else {
                 prepareChapterAdapter();
@@ -1446,7 +1444,7 @@ public class CustomPlaybackOverlayFragment extends Fragment implements LiveTvGui
     }
 
     private void prepareChapterAdapter() {
-        org.jellyfin.sdk.model.api.BaseItemDto item = mPlaybackController.getCurrentlyPlayingItem();
+        org.jellyfin.sdk.model.api.BaseItemDto item = playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem();
         List<ChapterInfo> chapters = item.getChapters();
 
         if (chapters != null && !chapters.isEmpty()) {

--- a/app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/LeanbackOverlayFragment.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/playback/overlay/LeanbackOverlayFragment.java
@@ -11,10 +11,9 @@ import org.jellyfin.androidtv.ui.playback.PlaybackController;
 import org.jellyfin.androidtv.ui.playback.PlaybackControllerContainer;
 
 import kotlin.Lazy;
+import timber.log.Timber;
 
 public class LeanbackOverlayFragment extends PlaybackSupportFragment {
-
-    private PlaybackController playbackController;
     private CustomPlaybackTransportControlGlue playerGlue;
     private VideoPlayerAdapter playerAdapter;
     private boolean shouldShowOverlay = true;
@@ -27,14 +26,17 @@ public class LeanbackOverlayFragment extends PlaybackSupportFragment {
         setBackgroundType(BG_LIGHT);
 
         PlaybackController playbackController = playbackControllerContainer.getValue().getPlaybackController();
+        if (playbackController == null) {
+            Timber.w("PlaybackController is null, skipping initialization.");
+            return;
+        }
 
         playerAdapter = new VideoPlayerAdapter(playbackController, this);
         playerGlue = new CustomPlaybackTransportControlGlue(getContext(), playerAdapter, playbackController);
         playerGlue.setHost(new CustomPlaybackFragmentGlueHost(this));
     }
 
-    public void initFromView(PlaybackController playbackController, CustomPlaybackOverlayFragment customPlaybackOverlayFragment) {
-        this.playbackController = playbackController;
+    public void initFromView(CustomPlaybackOverlayFragment customPlaybackOverlayFragment) {
         playerGlue.setInitialPlaybackDrawable();
         playerAdapter.setMasterOverlayFragment(customPlaybackOverlayFragment);
     }
@@ -69,7 +71,7 @@ public class LeanbackOverlayFragment extends PlaybackSupportFragment {
     }
 
     public void mediaInfoChanged() {
-        org.jellyfin.sdk.model.api.BaseItemDto currentlyPlayingItem = playbackController.getCurrentlyPlayingItem();
+        org.jellyfin.sdk.model.api.BaseItemDto currentlyPlayingItem = playbackControllerContainer.getValue().getPlaybackController().getCurrentlyPlayingItem();
         if (currentlyPlayingItem == null) return;
 
         playerGlue.invalidatePlaybackControls();

